admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Pod affinity_ and _pod anti-affinity_ allow you to specify rules about how pods should be placed relative to one another. For example, you could spread or pack pods within a service or relative to pods in other services or, you can make sure a pod is scheduled on a specific type of machine.
+
+Pod affinity/anti-affinity allow you to constrain which nodes your pod is eligible to be scheduled on based on the labels on other pods. A xref:../../architecture/core_concepts/pods_and_services.html#labels[label] is a key/value pair.
+
+* Pod affinity can tell the scheduler to locate a new pod on the same node as other pods if the label selector in the new pod matches the label in the current pod.
+* Pod anti-affinity can prevent the scheduler from locating a new pod on the same node as pods with the same labels if the label selector in the new pod matches the label in the current pod.
+
+There are currently two types of pod affinity rules, _required_ and _preferred_.
+
+Required rules *must* be met before a pod can be scheduled on a node. Preferred rules specify that, if the rule is met, the scheduler will try to enforce the rules, but will not guarantee enforcement.
+
+
+[[admin-guide-sched-affinity-pod-config]]
+== Configuring Pod Affinity/Anti-Affinity
+
+You configure pod affinity/anti-affinity through the pod specification files. You can specify a xref:admin-guide-sched-affinity-config-req[required rule], a xref:admin-guide-sched-affinity-config-pref[preferred rule], or both. If you specify both, the node must first meet the required rule, then meet the preferred.
+
+The following example shows a pod specification configured for pod affinity and anti-affinity.
+
+In this example, the pod affinity rule indicates that the pod can schedule onto a node only if that node has at least one already-running pod with a label that has the key `security` and value `S1`. The pod anti-affinity rule says that the pod prefers to not schedule onto a node if that node is already running a pod with label having key `security `and value `S2`.
+
+.Example pod config file with pod affinity
+====
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-pod-affinity
+spec:
+  affinity:
+    podAffinity: <1>
+      requiredDuringSchedulingIgnoredDuringExecution: <2>
+      - labelSelector:
+          matchExpressions:
+          - key: security <3>
+            operator: In <4>
+            values:
+            - S1 <3>
+        topologyKey: failure-domain.beta.kubernetes.io/zone
+  containers:
+  - name: with-pod-affinity
+    image: gcr.io/google_containers/pause:2.0
+
+----
+====
+<1> Stanza to configure pod affinity.
+<2> Defines a required rule.
+<3> The key and value (label) that must be matched to apply the rule.
+<4> The xref:../../rest_api/openshift_v1.html#unversioned-labelselectorrequirement[operator] represents the relationship between the key and a set of values.  Can be `In`, `NotIn`, `Exists`, or `DoesNotExist`.
```

Comment:
sorry. my misleading. 
I think the operator is self-explained, no need to make a translation. 
and for Exists and DoseNotExist, do not use values at all. 
It looks fine for me if only tell user the supported operators.